src/inputs/Table/Table.js

```diff
@@ -265,15 +318,24 @@ Table.propTypes = {
     label: PropTypes.string,
     loading: PropTypes.string,
     errorsPanelMainError: PropTypes.string,
+    placeholderTitle: PropTypes.string,
+    placeholderBody: PropTypes.string,
+    import: PropTypes.string,
+    export: PropTypes.string,
+    fullscreen: PropTypes.string,
   }),
   /**
    * Tooltip text
    */
   tooltipText: PropTypes.string,
   /**
-   *  List of extra React elements to add in the Table toolbar
+   *  List of extra React elements to add at top of TableToolbar
+   */
+  extraToolbarActions: PropTypes.array,
```

Comment:
This prop should not be changed for backward compatibility, do we have to remove `arrayOf(PropTypes.node)` from the propType for some reason?

Reply:
DONE